adapters/thetradedesk/thetradedesk_test.go

```diff
@@ -0,0 +1,353 @@
+package thetradedesk
+
+import (
+	"encoding/json"
+	"net/http"
+	"testing"
+
+	"github.com/prebid/openrtb/v20/openrtb2"
+	"github.com/prebid/prebid-server/v2/adapters"
+	"github.com/prebid/prebid-server/v2/adapters/adapterstest"
+	"github.com/prebid/prebid-server/v2/config"
+	"github.com/prebid/prebid-server/v2/openrtb_ext"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestJsonSamples(t *testing.T) {
+	bidder, err := Builder(
+		openrtb_ext.BidderTheTradeDesk,
+		config.Adapter{Endpoint: "https://direct.adsrvr.org/bid/bidder", ExtraAdapterInfo: "ttd"},
+		config.Server{ExternalUrl: "http://hosturl.com", GvlID: 1, DataCenter: "1"},
+	)
+	assert.Nil(t, err)
+	adapterstest.RunJSONBidderTest(t, "thetradedesktest", bidder)
+}
+
+func TestGetBidType(t *testing.T) {
```

Comment:
This should be covered under json unit test. Please remove this test.

Reply:
Is it harmful to leave these additional tests here? A few of them I find to be more scoped than the json